### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by anthony6401**
Fix some typos and add some descriptions.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -240,40 +248,49 @@ Lists all added tags.
...
 ![after](images/screenshots/list_command.png)
 
 ### Undoing command : `undo`
 
 Undo a recently executed command.<br>
 You can undo these commands: `tag`, `retag`, `untag`, `label`, `unlabel`, `clear`, and `redo`.
-<div markdown="block" class="alert alert-info">
-**Warning: Command history will be deleted once the app is closed!
+<div markdown="block" class="alert alert-warning">
+**warning: Command history will be deleted once the app is closed!
```
Comment (luo-git) [inline docs/UserGuide.md]: Missing `:`. Should be `**:warning`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -240,40 +248,49 @@ Lists all added tags.
...
 ![after](images/screenshots/undo_command.png)
 
 ### Redoing command : `redo`
 
 Redo a recently executed command.<br>
 Redo only exists when undo has been executed.
-<div markdown="block" class="alert alert-info">
-**Warning: Command history will be deleted once the app is closed!
+<div markdown="block" class="alert alert-warning">
+**warning: Command history will be deleted once the app is closed!
```
Comment (luo-git) [inline docs/UserGuide.md]: Same as the previous one.

**Review by luo-git — CHANGES_REQUESTED** [review]
Thank you for proof reading the UG! Apart from a small change, everything is good.👍

File: docs/UserGuide.md (hunk 1 above)
Comment (anthony6401) [reply docs/UserGuide.md]: Yeah, I miss that. Thank you for pointing it out!

**Review by luo-git — APPROVED** [review]
LGTM!👍

**Pull request merged**